src/Infrastructure/Symfony/Command/LoadFrCityDataCommand.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Infrastructure\Symfony\Command;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+#[AsCommand(
+    name: 'app:data:fr_city',
+    description: 'Load data into the fr_city table in the configured environment',
+    hidden: false,
+)]
+class LoadFrCityDataCommand extends Command
+{
+    public function __construct(
+        private EntityManagerInterface $em,
+        private readonly string $projectDir,
+    ) {
+        parent::__construct();
+    }
+
+    public function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $sql = file_get_contents($this->projectDir . '/data/fr_city.sql');
```

Comment:
Tu n'as normalement pas besoin du `projectDir`. `__DIR__` devrait suffire pour pouvoir remonter jusqu'au fichier sql.

Reply:
L'avantage c'est qu'on peut théoriquement pointer vers un autre fichier (pour des tests), ou encore pouvoir changer la commande d'endroit sans tout casser

Ça nous a paru plus solide que remonter par chemin relatif à partir de `__DIR__`

Reply:
Dans les faits, je ne pense pas que ce soit utile, mais bon, why not.